packages/react-components/src/components/resource-explorers/explorers/time-series-explorer/internal-time-series-explorer.tsx

```diff
@@ -56,7 +55,42 @@ export function InternalTimeSeriesExplorer({
   dropDownResourceDefinition = DEFAULT_TIME_SERIES_DROP_DOWN_DEFINITION,
   dropDownSettings: { isFilterEnabled: isDropDownFilterEnabled = false } = {},
   description = '',
+  timeZone,
+  significantDigits,
 }: TimeSeriesExplorerProps) {
+  const tableResourceDefinition =
```

Comment:
there is a Lot going on in this object definition, could it be its own function in a different file to clean things up?

Reply:
same comment for the internal-asset-property-explorer

Reply:
not a merge blocker though